pkg/gofr/container/mock_container.go

```diff
@@ -93,6 +94,9 @@ func NewMockContainer(t *testing.T, options ...options) (*Container, *Mocks) {
 	surrealMock := NewMockSurrealDB(ctrl)
 	container.SurrealDB = surrealMock
 
+	elasticSearchMock := NewMockElasticSearch(ctrl)
```

Comment:
This one remains, but I can live with it
 
```suggestion
	elasticsearchMock := NewMockElasticSearch(ctrl)
```

Reply:
sorry missed here. corrected now